monkey/infection_monkey/Pipfile

```diff
@@ -5,7 +5,6 @@ name = "pypi"
 
 [packages]
 altgraph = "*"
-cryptography = "==2.5" # We can't build 32bit ubuntu12 binary with newer versions of cryptography
```

Comment:
@shreyamalviya Have you tried actually building the monkey binaries with this change?

Reply:
I will do the tests now.

Reply:
No, that's the next task in the issue. I'll do it tomorrow if no one else does it by then.

Reply:
Linux 32bit failed...